prjxray-db/build.sh

```diff
@@ -8,10 +8,16 @@ if [ x"$TRAVIS" = xtrue ]; then
 fi
 
 DEVICES="artix7 kintex7 zynq7"
-DBDIR="${PREFIX}/share/symbiflow/prjxray-db"
+LOCAL_DBDIR="share/symbiflow/prjxray-db"
+DBDIR="${PREFIX}/${LOCAL_DBDIR}"
 
 mkdir -p $DBDIR
 
 for device in $DEVICES; do
     cp -r $device $DBDIR
 done
+
+DB_CONFIG="${PREFIX}/bin/prjxray-config"
+
+echo -e "#!/bin/bash\n\necho \`dirname \"\$0\"\`/../${LOCAL_DBDIR}" > ${DB_CONFIG}
```

Comment:
I believe you can just do "echo ${DBDIR}", and conda will take care of the rest.  See https://docs.conda.io/projects/conda-build/en/latest/resources/make-relocatable.html

Reply:
```
Any text file without NULL bytes that contains the build prefix or the placeholder prefix /opt/anaconda1anaconda2anaconda3 is registered in the info/has_prefix file in the package metadata. When conda installs the package, any files in info/has_prefix have the registered prefix replaced with the install prefix. For more information, see Package metadata.
```

Reply:
Right, done.